docker/build-tools/Dockerfile

```diff
@@ -271,6 +272,16 @@ RUN set -eux; \
     ${OUTDIR}/usr/local/google-cloud-sdk/bin/gcloud components install alpha --quiet; \
     rm -rf /usr/local/google-cloud-sdk/.install/.backup
 
+# Install cosign (for signing build artifacts) and verify signature
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN set -eux; \
+    ${OUTDIR}/usr/local/google-cloud-sdk/bin/gsutil -q cp gs://cosign-releases/${COSIGN_VERSION}/cosign-linux-amd64 /tmp/cosign \
```

Comment:
I know we have `TARGETARCH`  as well as checking for x86_64 and aarch64 in various places. This looks to limit us to and/x86_64. Is there an arm version of the code available?

Reply:
There is. I'm adding it.

Reply:
Actually, I take that back -- the only linux variant available is amd64 (see https://github.com/sigstore/cosign/releases/tag/v1.2.1). There's an amd64 and arm variant for mac, but that doesn't matter for this image afaik.

Reply:
Advantage to this is we can verify the binary before it's installed.